docs/ref/migration-operations.txt

```diff
@@ -185,15 +185,21 @@ defaults and always applies them in the Django ORM code.
 ``RemoveField``
 ---------------
 
-.. class:: RemoveField(model_name, name)
+.. class:: RemoveField(model_name, name, default=NOT_PROVIDED)
+
+.. versionchanged:: 5.1
+
+    Support for the ``default`` argument was added.
 
 Removes a field from a model.
 
 Bear in mind that when reversed, this is actually adding a field to a model.
 The operation is reversible (apart from any data loss, which is irreversible)
 if the field is nullable or if it has a default value that can be used to
 populate the recreated column. If the field is not nullable and does not have a
-default value, the operation is irreversible.
+default value, it will ask to provide a default value to populate existing
+rows and make the operation reversible. If the default value is not provided,
```

Comment:
```suggestion
default value, it will ask for a default value to populate existing
rows and make the operation reversible. If the default value is not provided,
```
This makes it a bit more concise and perhaps a bit easier to understand.